database/index.js

```diff
@@ -0,0 +1,25 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: '',
+  database: 'reviews',
+});
+connection.connect();
+
+const getMessages = (callback) => {
+  const query = 'SELECT * FROM messages WHERE listing_id = 50';
```

Comment:
Are you going to dynamically receive the listing id from the url? Just need to remember to remove the hard coded '50' at some point.

Reply:
Ya I will I just wanted some test data to mess with